kotlin/offline-routing/README.md

```diff
@@ -26,17 +26,14 @@ Tap once near a road to add a stop to the map. A number graphic will show its or
 1. Download the data from [ArcGIS Online](https://arcgisruntime.maps.arcgis.com/home/item.html?id=567e14f3420d40c5a206e5c0284cf8fc).
 2. Extract the contents of the downloaded zip file to disk.
 3. Open your command prompt and navigate to the folder where you extracted the contents of the data from step 1.
-4. Execute the following commands:
```

Comment:
I think this is handled in the big android Q update. If so, can we remove changes to this file?